p0_user_stories.md

```diff
@@ -31,22 +31,22 @@ other pods should be blocked from accessing it via any IP.
 
 - I want all namespaces matching X to be completely 100% locked-down by default; no pods can talk to any other pods until the developers specify policies allowing it - I may also want the converse situation for namespace Y.
 
-- I want to target all apps with network policies, but most of my apps need apiserver access, and i dont know the k8s service IP because all my clusters have different Service CIDRs.
+- I want to target all apps with network policies, but most of my apps need apiserver access, and i dont know the k8s service IP because all of my clusters have different Service CIDRs.
 
-- I want to block all pods from being able to reach ports on nodes. And the rule should automatically cover all nodes, and should cover all IPs on those nodes, except for port 53.
+- I want to block all pods from being able to reach ports on nodes. The rule should automatically cover all nodes, and all IPs on those nodes, except for port 53.
 
 ## tier-2
 
 - I want all namespaces to be “isolated” (don’t accept traffic from other Namespaces) by default; developers must specify if they want other namespaces to be able to access their services.
 
-- I want to administratively put a choke point between  pods that arent in the same app so i can audit cross-app dependencies and implement ingress controls by default in my cluster.
+- I want to administratively put a choke point between pods that arent in the same app so i can audit cross-app dependencies and implement ingress controls by default in my cluster.
 
 - I want to block all pods from being able to reach the AWS metadata server (169.254.169.254)
```

Comment:
This may be the same story as tier-1-story-2

How is this different than this [namespaceSelector option](https://kubernetes.io/docs/concepts/services-networking/network-policies/#behavior-of-to-and-from-selectors) ?